framework/core/src/Notification/UnsubscribeToken.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Notification;
+
+use Flarum\Database\AbstractModel;
+use Flarum\User\User;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class UnsubscribeToken extends AbstractModel
+{
+    /**
+     * The table associated with the model.
+     *
+     * @var string
+     */
+    protected $table = 'unsubscribe_tokens';
+
+    /**
+     * The attributes that should be cast to native types.
+     *
+     * @var array
```

Comment:
I think we should be able to remove the doc types/comments on these, since they should be inherited from the parent class?